packages/compass-generative-ai/src/components/ai-optin-modal.tsx

```diff
@@ -22,50 +22,42 @@ const GEN_AI_FAQ_LINK = 'https://www.mongodb.com/docs/generative-ai-faq/';
 type OptInModalProps = {
   isOptInModalVisible: boolean;
   isOptInInProgress: boolean;
+  isCloudOptIn: boolean;
   onOptInModalClose: () => void;
   onOptInClick: () => void;
   projectId?: string;
 };
 
-const titleStyles = css({
-  marginBottom: spacing[400],
-  marginTop: spacing[400],
-  marginLeft: spacing[500],
-  marginRight: spacing[500],
-  textAlign: 'center',
-});
-
-const bodyStyles = css({
-  marginBottom: spacing[400],
-  marginTop: spacing[400],
-  marginLeft: spacing[300],
-  marginRight: spacing[300],
-  display: 'flex',
-  flexDirection: 'column',
-  alignItems: 'center',
-  textAlign: 'center',
-});
-
 const disclaimerStyles = css({
   color: palette.gray.dark1,
-  marginTop: spacing[400],
-  marginLeft: spacing[800],
-  marginRight: spacing[800],
+  paddingLeft: spacing[800],
+  paddingRight: spacing[800],
 });
 
-const bannerStyles = css({
-  padding: spacing[400],
-  marginTop: spacing[400],
-  textAlign: 'left',
+// TODO: The LG MarketingModal does not provide a way to disable a button
+// so this is a temporary workaround to make the button look disabled.
+const disableOptInButtonStyles = css({
+  button: {
+    opacity: 0.5,
```

Comment:
Would we want to try to match the LG disabled styles, as in https://github.com/mongodb/leafygreen-ui/blob/f9f9ed34f21b561986c46c502cf13ef82a0e7c3c/packages/button/src/Button/Button.styles.ts#L360?

Using `opacity` here seems a bit odd and probably makes the button look a bit off compared to other disabled buttons?